Gems/Terrain/Assets/Materials/Terrain/TerrainDetailMaterial.materialtype

```diff
@@ -0,0 +1,142 @@
+            },
+            {
+                "$import": "../../../../Atom/Feature/Common/Assets/Materials/Types/MaterialInputs/SpecularPropertyGroup.json"
+            },
+            { 
+                "$import": "../../../../Atom/Feature/Common/Assets/Materials/Types/MaterialInputs/NormalPropertyGroup.json"
+            },
+            { 
+                "$import": "../../../../Atom/Feature/Common/Assets/Materials/Types/MaterialInputs/OcclusionPropertyGroup.json"
+            },
+            { 
+                "$import": "../../../../Atom/Feature/Common/Assets/Materials/Types/MaterialInputs/ParallaxPropertyGroup.json" 
+            },
+            {
+                "name": "terrain",
+                "displayName": "Terrain",
+                "description": "Terrain related properties",
+                "properties": [
+                    {
+                        "name": "overrideParallaxSettings",
+                        "displayName": "Override parallax settings",
+                        "description": "Override the height offset and bias from parallax",
+                        "type": "bool",
+                        "defaultValue": false
+                    },
+                    {
+                        "name": "heightOffset",
+                        "displayName": "Height offset",
+                        "description": "How much to offset the heights from 0.",
+                        "type": "float",
+                        "defaultValue": 0.0,
+                        "min": -1.0,
+                        "max": 1.0
+                    },
+                    {
+                        "name": "heightScale",
+                        "displayName": "Height scale",
+                        "description": "How much to scale the height map.",
+                        "type": "float",
+                        "defaultValue": 1.0,
+                        "min": 0.0,
+                        "max": 2.0
+                    },
+                    {
+                        "name": "blendFactor",
+                        "displayName": "Blend factor",
+                        "description": "Determines how hard the blend is between this material and other mateirals (lower is harder).",
+                        "type": "float",
+                        "defaultValue": 0.5,
+                        "min": 0.0,
+                        "max": 1.0
+                    },
+                    {
+                        "name": "weightClampFactor",
+                        "displayName": "Weight clamp factor",
+                        "description": "Determines the maximum weight value for when the height of the material begins to be clamped by the weight. For instance, a value of 0.1 means that when the weight is less than 0.1, the height will be reduced so the height is always 0 by the time the weight is 0.",
```

Comment:
Can this be split into 2 lines? It's pretty long.

Reply:
I don't believe that's allowed in json.